portal-bridge/src/api/execution.rs

```diff
@@ -303,25 +305,31 @@ impl ExecutionApi {
             .client
             .post("")
             .body_json(&json!(requests))
-            .map_err(|e| anyhow!("Unable to construct json post request: {e:?}"))?
+            .map_err(|e| anyhow!("Unable to construct json post request: {e:?}"))?;
```

Comment:
It's typically a good idea to not have multiple error messages that are the same (though ofc I know you didn't write these in this pr).
maybe something like `Unable to construct json post for batched requests: ...`